src/component/handlers/edit/editOnPaste.js

```diff
@@ -176,4 +186,15 @@ function insertFragment(
   );
 }
 
+function areTextBlocksAndClipboardEqual(
+  textBlocks: Array<string>,
+  blockMap: BlockMap
+): boolean {
+  let i = 0;
+  return (
+    textBlocks.length === blockMap.size &&
+    blockMap.toSeq().every((block) => block.getText() === textBlocks[i++])
```

Comment:
Ah, sorry -- I think you could do `blockMap.valueSeq().every(...)` to fix that. `toSeq` will keep the OrderedMap's keys, but I believe `valueSeq` should be zero-indexed.